arch/risc-v/src/common/riscv_exception_macros.S

```diff
@@ -0,0 +1,130 @@
+/****************************************************************************
+ * arch/risc-v/src/common/supervisor/riscv_exception_macros.S
```

Comment:
change riscv_exception_macros.S to riscv_macros.S since the macro is used in the normal context too.

Reply:
and remove supervisor/ from the path